src/routes/auth/verify.js

```diff
@@ -5,7 +5,7 @@ import { createSession, getUserByUsername, createUser } from '$lib/db';
 export async function post({ request }) {
   const { privateCode } = await request.json();
 
-  const res = await fetch('https://fluffyscratch.hampton.pw/auth/verify/v2/' + privateCode);
+  const res = await fetch('https://auth.itinerary.eu.org/api/auth/verifyToken?privateCode=' + privateCode = "&name=Aviate");
```

Comment:
You have to add `&name=Aviate` to the other API endpoint.

Reply:
I'll fix it.